tests/e2e/gc_metadata_syncer.go

```diff
@@ -1174,6 +1174,7 @@ var _ = ginkgo.Describe("[csi-guest] pvCSI metadata syncer tests", func() {
 		}()
 
 		var svClient clientset.Interface
+
```

Comment:
This file should not be part of the diff now

Reply:
done